src/server/routes/admin/nonces.ts

```diff
@@ -0,0 +1,169 @@
+import { Static, Type } from "@sinclair/typebox";
+import { FastifyInstance } from "fastify";
+import { StatusCodes } from "http-status-codes";
+import { eth_getTransactionCount, getRpcClient } from "thirdweb";
+import {
+  lastUsedNonceKey,
+  recycledNoncesKey,
+  sentNoncesKey,
+  splitLastUsedNonceKey,
+} from "../../../db/wallets/walletNonce";
+import { getChain } from "../../../utils/chain";
+import { normalizeAddress } from "../../../utils/primitiveTypes";
+import { redis } from "../../../utils/redis/redis";
+import { thirdwebClient } from "../../../utils/sdk";
+import { standardResponseSchema } from "../../schemas/sharedApiSchemas";
+import { walletWithAddressParamSchema } from "../../schemas/wallet";
+
+export const responseBodySchema = Type.Object({
+  result: Type.Array(
+    Type.Object({
+      walletAddress: Type.String({
+        description: "Backend Wallet Address",
+        examples: ["0xcedf3b4d8f7f1f7e0f7f0f7f0f7f0f7f0f7f0f7f"],
+      }),
+      onchainNonce: Type.String({
+        description: "Last mined nonce",
+        examples: ["0"],
+      }),
+      lastUsedNonce: Type.String({
```

Comment:
Nit: "lastUsed" is a bit ambiguous. Maybe:
- lastAcquiredNonce?
- dbNonce?
- engineNonce? (as opposed to onchainNonce)

Reply:
agreed, it's just the redis terminology + it's an admin route so wanted to keep it consistent between the two
would prefer a better name if breaking consistency between redis <> response is okay